development/compiling/compiling_for_linuxbsd.rst

```diff
@@ -128,6 +128,19 @@ manager.
           :ref:`doc_data_paths_self_contained_mode` by creating a file called
           ``._sc_`` or ``_sc_`` in the ``bin/`` folder.
 
+Compiling a headless/server build
+---------------------------------
+
+To compile a *headless* build which provides editor functionality to export
+projects in an automated manner, use::
+
+    scons -j8 platform=server target=release_debug
```

Comment:
I'd make `tools=yes` explicit. Also previous commands use `user@host:~/godot$` as prompt so it would be worth to use it to stay consistent (or remove it everywhere).

Reply:
I removed the prompt everywhere for consistency. This makes it easier to copy-paste commands as you can now triple-click them to select the whole line (without selecting the prompt at the same time).